starters/next-wordpress-starter/pages/examples/auth-api.jsx

```diff
@@ -0,0 +1,101 @@
+import Head from "next/head";
+import Link from "next/link";
+import { gql } from "@pantheon-systems/wordpress-kit";
+import { client } from "../../lib/WordpressClient";
+import { getFooterMenu } from "../../lib/Menus";
+import Layout from "../../components/layout";
+
+export default function AuthApiExampleTemplate({ menuItems, SSRPosts }) {
+  return (
+    <Layout footerMenu={menuItems}>
+      <Head>
+        <title>API Authorization Example</title>
+        <meta name="description" content="Generated by create next app" />
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+      <div className="prose lg:prose-xl mt-10 flex flex-col mx-auto max-h-screen">
+        <h1>API Authorization Example</h1>
+
+        <Link passHref href="/">
+          <span className="w-full underline cursor-pointer">Home &rarr;</span>
+        </Link>
+
+        <div className="mt-12 max-w-lg mx-auto lg:grid-cols-3 lg:max-w-screen-lg">
+          {SSRPosts?.length > 0 ? (
+            <p>
+              🎉 Next.js was able to successfully make an authenticated request
+              to Wordpress! 🎉
+            </p>
+          ) : (
+            <>
+              <p>
+                Next.js was unable to make an authorized request to the
+                Wordpress API. Please check your .env.development.local file to
+                ensure that your WP_APPLICATION_USERNAME and
+                WP_APPLICATION_PASSWORD are set correctly.
+              </p>
+              <p>
+                For more information on how to set these values, please see{" "}
+                <a href="https://github.com/pantheon-systems/decoupled-kit-js/blob/canary/web/docs/Frontend%20Starters/Next%20Wordpress/setting-environment-variables.md">
```

Comment:
```suggestion
                <a href="https://github.com/pantheon-systems/decoupled-kit-js/blob/canary/web/docs/Frontend%20Starters/Next.js/Next.js%20%2B%20WordPress/setting-environment-variables.md">
```
This link needs to be updated

Reply:
@lotusbaba this link is still incorrect. See the suggested change